dataprofiler/data_readers/data_utils.py

```diff
@@ -239,17 +268,17 @@ def read_json(data_generator, selected_columns=None, read_in_string=False):
         k += 1
     if not lines and k:
         raise ValueError("No JSON data could be read from these data.")
-    return lines
+    return cast(List[Dict], lines)
```

Comment:
as above, can we define it ahead of time instead of casting?

Reply:
Same as above.